stacks/BasicApiStack.ts

```diff
@@ -27,6 +26,12 @@ export function BasicApiStack ({ app, stack }: StackContext): { queue: Queue, bu
     }
   })
 
+  const bucket = new Bucket(stack, 'Car', {
+    notifications: {
+      topic: s3Topic
+    }
```

Comment:
are you planning on sending all S3 event types to the topic, or just the ones related to adding content to the bucket?

Reply:
I'll restrict it to just `ObjectCreated` events for now. We can expand that in the future if we need it.